stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -88,7 +93,10 @@ public void create(
                         protected ResponseWrapper doInBackground(Void... params) {
                             try {
                                 RequestOptions requestOptions =
-                                        RequestOptions.builder(publishableKey).build();
+                                        RequestOptions.builder(
+                                                publishableKey,
+                                                stripeAccount,
```

Comment:
@bdorfman-stripe -- the equivalent change for Token requests is fed in here, since (historically) the `RequestOptions` object (where various header information items are stored) was instantiated here instead of in the `StripeApiHandler` class. Some of this will be coming together with the 5-0 release.